src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,7 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
-    private String model;
+    String model;
```

Comment:
no, we should create setters and getter and make field private

Reply:
done